src/test/java/tutorspet/logic/parser/EditLessonCommandParserTest.java

```diff
@@ -35,7 +37,8 @@
     private static final Index VALID_MODULE_CLASS_INDEX_ONE = INDEX_FIRST_ITEM;
     private static final Index VALID_LESSON_INDEX_TWO = INDEX_SECOND_ITEM;
     private static final String INPUT_PREAMBLE =
-            " c/" + VALID_MODULE_CLASS_INDEX_ONE.getOneBased() + " l/" + VALID_LESSON_INDEX_TWO.getOneBased();
+            " " + PREFIX_CLASS_INDEX + VALID_MODULE_CLASS_INDEX_ONE.getOneBased()
+            + " " + PREFIX_LESSON_INDEX + VALID_LESSON_INDEX_TWO.getOneBased();
```

Comment:
Similar to the above comment, would it be better to format the statement as follows?
```suggestion
    private static final String INPUT_PREAMBLE = " "
            + PREFIX_CLASS_INDEX + VALID_MODULE_CLASS_INDEX_ONE.getOneBased() + " "
            + PREFIX_LESSON_INDEX + VALID_LESSON_INDEX_TWO.getOneBased();
```